libraries/AP_RCProtocol/AP_RCProtocol.h

```diff
@@ -195,6 +202,13 @@ class AP_RCProtocol {
         return _detected_with_bytes;
     }
 
+    // handle mavlink radio
+    void handle_radio_rc_channels(const mavlink_radio_rc_channels_t* packet);
+    struct {
+        mavlink_radio_rc_channels_t rc_channels;
```

Comment:
This seems to be chewing a chunk of ram per instance? We have a lot of instances

Reply:
it is located in the frontend, so there should be "just" one of them, as I understand it.
It is 66 bytes. I could make it a pointer and allocate as needed as you suggested elsewhere before, or locate it in the AP_RCProtocol_MAVLinkRadio backend driver and provide a method to handle it (maybe nicier), if you want to.